auth_backend/modules/user/listeners.py

```diff
@@ -0,0 +1,17 @@
+from auth_backend.modules.common import utils as common_utils
+
+
+def send_invite_mail(sender, instance, created, **kwargs):
```

Comment:
is this properly working as a listener? I thought the syntax is different - might be wrong tho